src/fss/dif.rs

```diff
@@ -115,27 +115,27 @@ pub fn h(prg: &mut impl PRG, seed: u128) -> CorrectionWord {
     let z_l = out[2] as u32;
     let z_r = (out[2] >> 32) as u32;
 
-    // TODO: A 3x expansion PRG is slightly overkill
+    // TODO: A 3x expansion Prg is slightly overkill
     let t_l = (out[2] >> 64) as u8 & 1u8;
     let u_l = (out[2] >> 65) as u8 & 1u8;
     let t_r = (out[2] >> 66) as u8 & 1u8;
     let u_r = (out[2] >> 67) as u8 & 1u8;
 
     CorrectionWord {
-        s_l,
-        t_l,
         z_l,
         u_l,
-        s_r,
-        t_r,
+        s_l,
+        t_l,
         z_r,
         u_r,
+        s_r,
+        t_r,
```

Comment:
Why do you need to reorder the struct fields? That's fine since we access by name but I'm curious